packages/tc-ui/src/pages/view/template.jsx

```diff
@@ -96,24 +96,23 @@ const View = props => {
 					</div>
 					<div className="o-layout-typography">
 						{Object.entries(schema.fieldsets).map(
-							([name, { heading, properties }]) => (
-								<section
-									className={`fieldset-treecreeper fieldset-${name}`}
+							([name, { heading, description, properties }]) => (
+								<Fieldset
```

Comment:
Is there a way you can force the Fieldset component to use the `<section>` html tag on the view page? `<fieldset>` shouldn't really be used outside of forms, for accessibility reasons

Reply:
Ahh ok I didn't know that. Good to know - so I have updated it to make it conditional on `includeEditLink` as that will only show on the view. Do you think it would benefit from being based on a more explicit prop - like `displayOnForm` or something?

Reply:
Maybe something like `isInEditMode`, which acts as the trigger for all the differences in output between edit and view